### DISCUSSION TITLE
what happened to the releases page?

### DISCUSSION THREAD

**Discussion by ralyodio** [rabbitmq/rabbitmq-delayed-message-exchange#278] [General]
### Is your feature request related to a problem? Please describe.

what happened to the releases page? 

### Describe the solution you'd like

what happened to the releases page? 

### Describe alternatives you've considered

what happened to the releases page?

**Comment by michaelklishin** [comment]
@ralyodio that's an extremely terse "problem definition". Releases of this plugin have been published to [GitHub releases](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases) in this repo for years.

**Comment by ralyodio** [comment]
sorry, i guess github moved their releases page -- it used to be a top level tab and i didn't see it.
